src/DynamoCoreWpf/ViewModels/PackageManager/PackagePathViewModel.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using Dynamo.Interfaces;
+using DelegateCommand = Dynamo.UI.Commands.DelegateCommand;
+
+
+namespace Dynamo.ViewModels
+{
+    class PackagePathViewModel : ViewModelBase
+    {
+        public ObservableCollection<string> RootLocations { get; private set; }
+        public int SelectedIndex { get; set; }
```

Comment:
In the setter of `SelectedIndex`, there should be multiple calls to `RaiseCanExecuteChanged` on the relevant command. For example, when the `SelectedIndex` is on the first or last item, move up/down will be disabled accordingly (can't move the selection item higher/lower).